appstreamcli thinks ca and ca@valencia is the
same and fails to validate

--- a/shell/appdata/meson.build
+++ b/shell/appdata/meson.build
@@ -15,7 +15,7 @@ appdata_file = i18n.merge_file(
 
 # Validate Appdata
 appstreamcli = find_program('appstreamcli', required: false)
-if appstreamcli.found()
+if false
   test(
     'validate-appdata',
     appstreamcli,
